src/ServicePulse.Host/app/img/monitoring-lost.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file needs to be added to the csproj file. I think the release build might break unless this is included in the csproj